index.js

```diff
@@ -100,23 +129,31 @@ function myMap(array, callback) {
 // ***** Callbacks - Question 1 *****
 
 // *** Uncomment the lines below to test
-// console.log("%cCallbacks - Question 1", "color: red")
+console.log("%cCallbacks - Question 1", "color: red")
+
+function triple(threeTimes) {
+  console.log(threeTimes * 3)
+}
```

Comment:
Nice! Only issue is that you're console logging instead of returning. It's important for this function to return something because the myMap function is expecting back some return value. Here's the solution:

```javascript
function triple(num) {
  return threeTimes * 3
}
```